client/app/components/organizations/handlers/show_categories.js

```diff
@@ -20,9 +20,9 @@ export default React.createClass({
     var unique = {}
 
     for( var i in organizations ){
-       if( typeof(unique[organizations[i].category]) == "undefined" && organizations[i].category !== undefined){
-        distinctCategories.push(organizations[i].category);
-       }
+      if( (typeof(unique[organizations[i].category]) == "undefined" && organizations[i].category !== undefined && organizations[i].status === "active") ){
```

Comment:
This is inconsistent when I refresh the page. There were two categories that should have showed up in testing, and sometimes it worked, and then sometimes it would only show one. Weird. Needs some tweeking.